### DISCUSSION TITLE
Is it possible to write custom functionality in app/* ?

### DISCUSSION THREAD

**Discussion by zssvaidar** [atrocore/atrocore#514] [Q&A] [answered]
We want to store `custom modules` or write custom functionality in root as `app/` directory? Can you elaborate on possible implementations*?

**Comment by rratsun** [comment, marked as answer]
Hello,

You can develop your own module. How to do it you can find [here](https://help.atrocore.com/#/atrocore/developer-guide/creating-own-module).
